tests/D2L.CodeStyle.TestAnalyzers.Tests/NUnit/CategoryAnalyzerTests.cs

```diff
@@ -0,0 +1,176 @@
+﻿using D2L.CodeStyle.TestAnalyzers.Common;
+using D2L.CodeStyle.TestAnalyzers.Test.Verifiers;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using NUnit.Framework;
+
+namespace D2L.CodeStyle.TestAnalyzers.NUnit {
+	[TestFixture]

```

Comment:
These tests are incomplete, but all the copy-pasta and changing line/column numbers and stuff was uninteresting :(